ip_link_show.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: gather ip link addresseses
```

Comment:
I would suspect we get this data already if we would gather facts? Why can't we use that one?

Reply:
I believe this script's intent was to the get the mac address, i have updated the overall name field.